app/main.py

```diff
@@ -1,8 +1,45 @@
-class Car:
-    # write your code here
-    pass
+from typing import List
+
 
+class Car:
+    def __init__(self, comfort_class: int, clean_mark: float, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: float, average_rating: float, count_of_ratings: int):
```

Comment:
Add a return annotation (None) and wrap the arguments on new lines.
![Uploading Screenshot 2023-07-23 at 13.24.44.png…]()